tests/test_post_refactor_cleanup.py

```
@@ -0,0 +1,112 @@
+import warnings
+before = list(warnings.filters)
+import agentor  # must not touch the global filter list
+added = [f for f in warnings.filters if f not in before]
+print("ADDED", added)
+
+shown = []
+warnings.showwarning = lambda *a, **k: shown.append(a[0])
+warnings.warn("host application deprecation", DeprecationWarning)
+print("SHOWN" if shown else "SWALLOWED")
+"""
+    result = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)